index.html

```diff
@@ -8,37 +8,40 @@
 </head>
 
 <body>
-    <div class="menu">
-        <div id="mleft">
-            Just my another site
-        </div>
-        <div id="mright">
-            <a class="button">
+    <div class="part menu">
```

Comment:
Tak jak w Food is Good dodałem part i limit, ponieważ trzeba ograniczyć stronę, jeżeli chce się mieć przestrzeń po bokach. Limit jest środkowany przez margin: 0 auto(zobacz css), ponieważ ma ustaloną szerokość.